xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/dashboard/gadgetWizard.js

```diff
@@ -68,8 +68,9 @@ define(['jquery', 'xwiki-ckeditor'], function($, ckeditorPromise) {
   };
 
   var getDefaultGadgetTitle = function(macroEditor) {
-    var gadgetName = macroEditor.attr('data-macroid').split('/')[0];
-    return "$services.localization.render('rendering.macro." + gadgetName + ".name')";
+    const gadgetName = macroEditor.attr('data-macroid').split('/')[0];
+    return '{{translation key="rendering.macro.' +
```

Comment:
Are those documents always xwiki/2.x syntax document by definition ?

Reply:
I've removed this change.

Reply:
@tmortagne It was more work than I initially thought, but the correct syntax is now available in the macro dialog. I've also implemented a simple logic to use the wiki macro when it's `xwiki/2.x` syntax. As a side effect, the widget content itself is now also stored with the document's actual syntax (it was always `xwiki/2.0` before).